modules/caddyhttp/matchers.go

```diff
@@ -986,6 +986,7 @@ func (mre *MatchRegexp) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 var wordRE = regexp.MustCompile(`\w+`)
 
 const regexpPlaceholderPrefix = "http.regexp"
+const MatcherErrorVarKey = "matchers.error"
```

Comment:
This should probably have a comment explaining what it is and how to use it.

Reply:
Right - will do

Reply:
Done!